CRM/Utils/JSON.php

```diff
@@ -39,16 +39,28 @@
 class CRM_Utils_JSON {
 
   /**
-   * Function to create JSON object
+   * Output json to the client.
+   * @param mixed $input
+   */
+  public static function output($input) {
+    header('Content-Type: application/json');
+    CRM_Utils_JSON::output($input);
```

Comment:
@eileenmcnaughton Whoops this is an infinite loop.
Funny because I made the same mistake when I first wrote this fn too :P